cmd/kafka-consumer/main.go

```diff
@@ -91,7 +91,7 @@ func init() {
 	flag.StringVar(&key, "key", "", "Private key path for Kafka SSL connection")
 	flag.Parse()
 
-	err := logutil.InitLogger(&logutil.Config{
+	err := logutil.InitLoggerForCDC(&logutil.Config{
```

Comment:
Should `pkg/cmd/util/helper.go` also needs to initialize logger this way?

Reply:
No. It doesn't  rely on `grpc` and `sarama` actually.

Reply:
Fix.